File: src/finance/trust-account-service/src/main/java/com/estateiq/trustaccountservice/Application.java
package com.estateiq.trustaccountservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;

/**
 * trust-account-service.
 *
 * <p>CLAUDE.md rule #6 — trust account ledger entries are append-only. Never UPDATE/DELETE.
 * Tenant deposits and client funds are managed in segregated accounts with a full audit
 * trail (RICS, REIA, UAE). Deploy strategy: blue-green only (no canary).
 */
@SpringBootApplication
@RestController
public class Application {
    public static void main(String[] args) { SpringApplication.run(Application.class, args); }

    @GetMapping("/healthz")
    public Map<String, String> health() {
        return Map.of("status", "ok", "service", "trust-account-service");
    }
}

File: src/valuation/avm-service/src/main/kotlin/com/estateiq/avmservice/Application.kt
package com.estateiq.avmservice

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@SpringBootApplication
@RestController
class Application {
    @GetMapping("/healthz")
    fun health(): Map<String, String> = mapOf("status" to "ok", "service" to "avm-service")
}

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
